mathgo/v4/mathgo.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"time"
+)
+
+const secondHandLength = 90
+const clockCentreX = 150
+const clockCentreY = 150
+
+type Point struct {
+	X float64
+	Y float64
+}
+
+func SecondHand(w io.Writer, t time.Time) Point {
```

Comment:
exported function without comment

Reply:
Understood. Will add block comments to exported functions.